InheritanceDemo.java

```diff
@@ -0,0 +1,30 @@
+package com.basicsstrong.oop;
+interface A{
+	default void method() {
+		System.out.println("This is interface A");
+	}
+}
+interface B{
+	default void method() {
+		System.out.println("This is interface B");
+	}
+}
+class C implements A,B{
+
+	@Override
+	public void method() {
```

Comment:
What happens if you don't override the method ? Which interfaces method will be called if you create an instance of class C and invoke method() on that instance ?

Reply:
We need to set default interface's method so we will get an error if we dont override the method. C class's method is called first and the one interface we set it as default interface's method is called